src/game/level.rs

```diff
@@ -0,0 +1,31 @@
+//! Setup the level
+
+use bevy::prelude::*;
+
+use super::physics::{PhysicalTransform, Velocity};
+use crate::screen::Screen;
+
+#[derive(Debug, Event)]
+pub(crate) struct SpawnLevel;
+
+pub(super) fn plugin(app: &mut App) {
+    app.observe(spawn_level);
+}
+
+fn spawn_level(
+    _trigger: Trigger<SpawnLevel>,
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+) {
+    commands.spawn((
+        Name::new("Player"),
+        SpriteBundle {
+            texture: asset_server.load("ducky.png"),
+            transform: Transform::from_scale(Vec3::splat(0.4)),
+            ..Default::default()
+        },
+        Velocity::default(),
+        PhysicalTransform::default(),
```

Comment:
This works here, but physics engines usually rely on `Transform` directly, would the solution be as following?
- Root entity -- has child -> Physical entity
- Root entity -- has child -> Graphical entity
- FixedUpdate -- update_physics()
- Update -- interpolate_graphics()

I don't think we should add this needless boilerplate here, but it's more than worth few lines of comments, since basically every 2D game, top down or platformer uses some sorts of physics

Reply:
Good point, I added a comment.
I did not rename `update_rendered_transform` to `interpolate_graphics` because the fact that it uses interpolation instead of advanced simulation or extrapolation is a detail.

Reply:
Yeah, the names aren't important at all here :)